source/billing/credit-card.rst

```diff
@@ -0,0 +1,120 @@
+###################
+Credit Card payment
+###################
+
+.. note::
+   Only users with the Project Admin role will be able to view and modify the billing details
+
+The Catalyst Cloud now accept payments via credit card. This section will explain how you
+can setup a credit card to pay your invoices.
+
+You and/or your billing contact will still receive a usage summary from us each month with your usage details and a confirmation that your invoice has been paid.
+
+Adding and switching to credit card payment
+===========================================
+
+To setup a credit card, please login_ to the dashboard and follow these steps:
+
+1) Go to Account Details, under Management
+2) Select the Project Billing Details tab
+
+.. image:: resources/billing-details.png
+   :align: center
+
+As you can see from the screenshot, your payment method should be as "Invoice".
+There are a few different ways to setup your credit card payment. You can also add a credit card without changing the payment method.
+
+To change to credit card and to set it up click on Change Payment method:
```

Comment:
To change your active payment method to a credit card, you will need to click on  *Change payment method*: